ginga/modes/contrast.py

```diff
@@ -0,0 +1,128 @@
+#
+# contrast.py -- mode for setting contrast
+#
+# This is open-source software licensed under a BSD license.
+# Please see the file LICENSE.txt for details.
+#
+"""Contrast Mode enables bindings that can adjust the contrast of
+an image in a Ginga image viewer.
+
+Enter the mode by
+-----------------
+* Space, then "t"
+
+Exit the mode by
+----------------
+* Esc
+
+Default bindings in mode
+------------------------
+* T : restore contrast to defaults
+* left drag : adjust contrast
+    Interactive shift/stretch colormap (aka contrast and bias).
+    Moving left/right controls shift, up/down controls stretch.
+    Release button when satisfied with the contrast.
```

Comment:
Line breaks not rendered properly. Try this, maybe?
```suggestion
    * Interactive shift/stretch colormap (aka contrast and bias).
    * Moving left/right controls shift, up/down controls stretch.
    * Release button when satisfied with the contrast.
```